cltk/corpus/odia/alphabet.py

```diff
@@ -0,0 +1,26 @@
+'''Odia is an Indian language, belonging to the Indo-Aryan branch of
+the Indo-European language family.It is mainly spoken in the Indian
+states of Odisha and in parts of West Bengal, Jharkhand, Chhattisgarh
+and Andhra Pradesh.Odia is one of the many official languages in India;
+it is the official language of Odisha and the second official language
+of Jharkhand.Odia is the predominant language of Odisha, where Odia speakers comprise
+around 83.33% of the population according to census surveys.
+'''
+VOWELS = [
+    'ଅ', 'ଆ', 'ଇ', 'ଈ', 'ଉ', 'ଊ', 'ଋ',
+    'ୠ', 'ଌ', 'ୡ', 'ଏ', 'ଐ', 'ଓ', 'ଔ']
+
+STRUCTUREDCONSONENTS = [
```

Comment:
Typo? (STRUCTUREDCONSONENTS -> STRUCTUREDCONSONANTS)

Reply:
Yes. I'll make the changes .